graylog2-web-interface/src/components/welcome/Welcome.jsx

```diff
@@ -33,7 +33,13 @@ const StyledSectionComponent = styled(SectionComponent)`
 const Welcome = () => (
   <>
     <PageHeader title="Welcome to Graylog">
-      <span>Here you can find most used content</span>
+      <span>
+        Here you can find most used content.
+        <br />
+        Last open - entities which were recently viewed.
+        Favorite items - entities which were added to the favorites.
+        Recent activity - shows all actions ( creating, updating, sharing e.t.c) which were made with all entities.
+      </span>
```

Comment:
What do you think about displaying the description per section, below the section headline?

Regarding "last opened" and "favorite", in my opinion it makes sense to mention which entities are included in the lists. Currently it only contains saved searches and dashboards. It means we need to update the text when we extend the functionality, but I like that it gives users a clear understand of where / when the functionality is offered.

Last open
Overview of recently visited saved searches and dashboards.

Favorite items
Overview of your favorite saved searches and dashboards.

Recent activity
{if admin} This list includes all actions graylog users performed, like creating or sharing an entity.
{if non-admin user} Overview of actions you made with entities, like creating or sharing an entity.


Regarding "recent activity", it looks only an admin can see "creating", "updating" and "deleting" and anormal user just sees the activity for sharing, but I think we will change this soon.